andela/gimli-cfh
# Invite friends to game

What does this PR do?

This PR enables players to send in app invite to friends.

Description of Task to be completed?

A game player should send in app invite to other players saved as friends to play game with.

How should this be manually tested?

Click the add friends button to test

Any background context you want to provide?

N/A

What are the relevant pivotal tracker stories?

#145026997 - Registered users should be able to invite other users to connect as "friends"

Screenshots (if appropriate)

N/A

Questions:

N/A
